Kudu.Core.Test/HgRepositoryFacts.cs

```diff
@@ -17,7 +17,7 @@ public void FetchWithoutConflictsRetriesWithRecoveryIfInitialFetchFails()
             // Arrange
             var executable = new Mock<IExecutable>();
             executable.Setup(e => e.EnvironmentVariables).Returns(new Dictionary<string, string>());
-            executable.Setup(e => e.Execute(It.IsAny<ITracer>(), "pull {0} --branch {1} --noninteractive", It.IsAny<object[]>()))
+            executable.Setup(e => e.Execute(It.IsAny<ITracer>(), "pull '{0}' --branch '{1}' --noninteractive", It.IsAny<object[]>()))
```

Comment:
single quote works for Windows?   I would think it has to be double quotes on command line.